extensionsGallery-insider.json

```diff
@@ -1,386 +1,459 @@
-								"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/samples/serverReports/package.json"
-							},
-							{
-								"assetType": "Microsoft.VisualStudio.Services.Content.License",
-								"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/samples/serverReports/LICENSE.txt"
-							}
-						],
-						"properties": [
-						{ "key": "Microsoft.VisualStudio.Code.ExtensionDependencies", "value":""},
-						{ "key": "Microsoft.VisualStudio.Code.Engine", "value":"*"},
-						{ "key": "Microsoft.VisualStudio.Services.Links.Source", "value":"https://github.com/Microsoft/sqlopsstudio"}
-						]
-					}
-				],
-				"statistics": [],
-				"flags": "preview"
-			},
-			{
-				"extensionId": "13",
-				"extensionName": "ssmskeymap",
-				"displayName": "SSMS Keymap",
-				"shortDescription": "This extension ports popular SSMS keyboard shortcuts to SQL Operations Studio",
-				"publisher": {
-					"displayName":"Kevin Cunnane",
-					"publisherId": "kevcunnane",
-					"publisherName":"kevcunnane"
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/ServerReports/server-report-0.1.3.vsix"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/samples/serverReports/images/sqlserver.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/samples/serverReports/README.md"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Code.Manifest",
+									"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/samples/serverReports/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/master/samples/serverReports/LICENSE.txt"
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": "*"
+								},
+								{
+									"key": "Microsoft.VisualStudio.Services.Links.Source",
+									"value": "https://github.com/Microsoft/sqlopsstudio"
+								}
+							]
+						}
+					],
+					"statistics": [],
+					"flags": "preview"
 				},
-				"versions": [
-					{
-						"version": "0.2.0",
-						"lastUpdated": "4/16/2018",
-						"assetUri": "",
-						"fallbackAssetUri": "fallbackAssetUri",
+				{
+					"extensionId": "13",
+					"extensionName": "ssmskeymap",
+					"displayName": "SSMS Keymap",
+					"shortDescription": "This extension ports popular SSMS keyboard shortcuts to SQL Operations Studio",
```

Comment:
Minor: want to fix this naming too? I'll update the Github project to match. Maybe I should find new shortcuts to add with the rename 😄